lib/facebook/messenger/server.rb

```diff
@@ -122,7 +144,7 @@ def trigger_events(events)
         events['entry'.freeze].each do |entry|
           # Facebook may batch several items in the 'messaging' array during
           # periods of high load.
-          entry['messaging'.freeze].each do |messaging|
+          entry['messaging'.freeze].try(:each) do |messaging|
```

Comment:
Aha, okay! I'll keep that in mind for other callbacks, then. Oh, and please feel free to make a pull request with that when you're done – I'm sure there are people who would love to have that upstream. :-)